tfexec/terraform_workspace_select.go

```diff
@@ -0,0 +1,19 @@
+package tfexec
+
+import (
+	"context"
+)
+
+//WorkspaceSelect selects the workspace "workspace". The workspace needs to exist
+// in order for the switch to be successful
+func (c *terraformCLI) WorkspaceSelect(ctx context.Context, workspace string, dir string) error {
```

Comment:
Can we remove the `dir` parameter as same as the `WorkspaceNew`?

Reply:
removed.